scripts/simulator_base.js

```diff
@@ -400,8 +400,6 @@ var SIMULATOR = {};
 				// Check Evade
 				if (target.invisible) {
 					target.invisible--;
-					// Missed - retry next turn
-					skill.countdown = 0;
```

Comment:
📝 This automatically happens in the calling function if we return `0` affected units.